### PR TITLE
module add

### PR DESCRIPTION
**Pull request by LittleBadBad**
added missing module ‘cover’

### CODE REVIEW

File: src/api/cover/models/Cover.settings.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "connection": "default",
+  "collectionName": "",
+  "info": {
+    "name": "cover",
+    "description": "封面"
```
Comment (Zayelion) [inline src/api/cover/models/Cover.settings.json:6]: Change this to English please

File: src/plugins/graphql/config/generated/schema.graphql [hunk 2]
```diff
@@ -142,6 +142,71 @@ input CommentInput {
   forum: ID
 }
 
+"""封面"""
```
Comment (Zayelion) [inline src/plugins/graphql/config/generated/schema.graphql:145]: Same here, switch it over to English

**Review by Zayelion — COMMENTED** [review]
Looks fine, just need you to fix some language.

**Comment by LittleBadBad** [comment]
Thanks, I'll modify later------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;Jamezs&amp;nbsp;Gladney&quot;<notifications@github.com&gt;
发送时间:&nbsp;2021年1月28日(星期四) 晚上10:49
收件人:&nbsp;&quot;SalvationDevelopment/YGOSalvation-Admin&quot;<YGOSalvation-Admin@noreply.github.com&gt;;
抄送:&nbsp;&quot;LittleBadBad&quot;<1287299719@qq.com&gt;;&quot;Author&quot;<author@noreply.github.com&gt;;
主题:&nbsp;Re: [SalvationDevelopment/YGOSalvation-Admin] module add (#10)

**Pull request merged**